build.sbt

```diff
@@ -135,6 +136,7 @@ lazy val persistence = (project in file("persistence"))
 lazy val query = (project in file("query"))
   .settings(hmdaBuildSettings:_*)
   .settings(
+    parallelExecution in Test := false,
```

Comment:
Yeah. This isn't related to the implementation of edits per se, but I noticed that we have been running tests sequentially by default, so I've [changed](https://github.com/cfpb/hmda-platform/pull/851/files#diff-c6575342216356c040d3b53a22f877c4R21) that. There are some exceptions though, since some tests expect to be run in a certain sequence (they usually are the ones where we're dealing with a database of some sort). This change should make your tests run much faster overall (though not in this particular project).